components/admin_console/reset_email_modal/reset_email_modal.jsx

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Modal} from 'react-bootstrap';
+import {FormattedMessage} from 'react-intl';
+
+import {adminResetEmail} from 'actions/admin_actions.jsx';
+import * as Utils from 'utils/utils.jsx';
+
+export default class ResetEmailModal extends React.Component {
+    static propTypes = {
+        user: PropTypes.object,
+        show: PropTypes.bool.isRequired,
+        onModalSubmit: PropTypes.func,
+        onModalDismissed: PropTypes.func,
+        passwordConfig: PropTypes.object,
+    };
+
+    static defaultProps = {
+        show: false,
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            error: null,
+        };
+    }
+
+    componentWillUnmount() {
+        this.setState({error: null});
```

Comment:
This is probably copied from another one of our modals, but I think we can eliminate this cleanup. From the conversation at https://github.com/facebook/react/issues/12111, 
> If the component is being unmounted, there's no need to "reset" the state. A component can't be "resurrected" so if it gets unmounted, its state is irrelevant to the rest of the program.